app/components/QrCodeScanner/QrCodeScanner.jsx

```diff
@@ -0,0 +1,66 @@
+// @flow
+import React, { Component } from 'react'
+
+import Button from '../Button'
+import Instascan from 'instascan'
+
+import qrCodeScannerStyles from './QrCodeScanner.scss'
+
+type Props = {
+  callback: Function,
+}
+
+type State = {
+  scanner: Object
+}
+
+class QrCodeScanner extends Component<Props, State> {
+  scanPreviewElement: HTMLVideoElement | null
+
+  state = {
+    scanner: {}
```

Comment:
I think `scannerInstance` is a better name

Reply:
updated